examples/readme.md

```diff
@@ -0,0 +1 @@
+The examples moved to [codepen](http://codepen.io/collection/nwWJbe/).
```

Comment:
@vvo are there any good example on how to use this new feature? Can't find any examples related to this.